src/lxc/conf.h

```diff
@@ -150,14 +150,18 @@ struct lxc_tty_info {
  * optionals pivot_root, rootfs mount paths
  * @path       : the rootfs source (directory or device)
  * @mount      : where it is mounted
- * @options    : mount options
  * @bev_type   : optional backing store type
+ * @options    : mount options
+ * @mountflags : the portion of @options that are flags
```

Comment:
I wonder if it makes sense to add these as part of a larger commit like this without mentioning it in the commit message. I currently only see them parsed out but not actually used?